salt/runners/auth.py

```diff
@@ -20,6 +20,10 @@ def mk_token(**load):
     '''
     Create an eauth token using provided credentials
 
+    Non-root users may specify an expiration date (if allowed via the
+    :conf_master:`token_expire_user_override` setting) by passing an additional
+    ``token_expire`` param.
```

Comment:
It's not clear from reading this what the parameter should *be* though. Below I see an example of it being `94670856` but I have no idea what this means. Could we clarify this a bit, perhaps with an example and explanation?